lte/gateway/c/oai/tasks/s1ap/s1ap_state_manager.cpp

```diff
@@ -165,12 +165,6 @@ void S1apStateManager::clear_s1ap_imsi_map() {
   hashtable_uint64_ts_destroy(s1ap_imsi_map_->mme_ue_id_imsi_htbl);
 
   free_wrapper((void**) &s1ap_imsi_map_);
-
-  if (persist_state_enabled) {
-    std::vector<std::string> keys_to_del;
-    keys_to_del.emplace_back(S1AP_IMSI_MAP_TABLE_NAME);
-    redis_client->clear_keys(keys_to_del);
-  }
 }
```

Comment:
is `S1apStateManager::clear_s1ap_imsi_map()` called only during the service exit?

Reply:
Yes, it is only called at service exit, to clear all the memory allocated to the hashtable.